### ISSUE TITLE
Address bar is needlessly and sometimes problematically small on tiled Nemo; or, are icons missing in tiled Nemo?

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/nemo#2942]
```
 * Nemo version (nemo --version) 5.2.4
 * Is issue with desktop or windowed nemo? Windowed.
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 20.3 Cinnamon.
 * Graphics hardware *and* driver used: Intel; Mesa
 * 32 or 64 bit: 64
 ```

**Issue**

The address bar is needlessly and sometimes problematically small on tiled Nemo.

Untiled / maximised Nemo:

![image](https://user-images.githubusercontent.com/47056543/157996031-12caa94d-b574-436b-9538-3981fc139b90.png)

Tiled Nemo:

![image](https://user-images.githubusercontent.com/47056543/157996086-5e9b126b-be07-4891-935b-ec4f25c15666.png)

For short paths - as for the `/` show in the screenshots - the short address bar is not a problem, but it is a problem for longer paths, though the options to show path in the title bar does mitigate.

**Steps to reproduce**

1 Have Nemo maximised.
2 Tile Nemo to the right or to the left.

**Expected behaviour**

The address bar should use pretty much all of the room that it can use.

Actually, perhaps the problem here is that, when tiled, some Nemo icons are missing. It is as if the tiled window leaves room for those icons - icons visible, in the maximised window, to the right of the address bar - but does not actually draw them.

**Other information**

I am using x2 DPI, on a screen that, when unscaled, has these dimensions: 2560 x 1440.

**Comment by LinuxOnTheDesktop** [comment]
This problem persists under Mint 21, on Nemo 5.4.3. Screenshot:

![image](https://user-images.githubusercontent.com/47056543/201729296-68da131b-722c-415c-a944-4d28c24f8254.png)

**Comment by Drugwash2** [comment]
No, actually what should be happening - if the devs had any vision whatsoever - is to horizontally shrink those ridiculously wide arrows to the left, gather all buttons in a single row, shrink the buttons row vertically and move the address bar below the buttons row, perhaps also slightly shrinked vertically.
Maybe you could get some inspiration from here:
![Screenshot from 2022-11-15 17-55-35](https://user-images.githubusercontent.com/29117024/201965923-355d769c-9390-41fb-9745-c55ccb41a55f.png)

**Comment by LinuxOnTheDesktop** [comment]
@Drugwash2 : I suspect that you are enlarging the scope of this bug report; certainly you add counter-productive vitriol,

@mtwebster: thank you for the attention. I am unsure that I understand what you write. Are you proposing to make a change such that, though the input box does not get larger, the space next to the box becomes populated and, thus, the user has some explanation of the smallness of the input box? If so, then that explanation will at least sugar the pill (as it were) of the small input box; as things stand, the smallness of the box is especially irking because it looks gratuitous.

**Comment by Drugwash2** [comment]
>  I suspect that you are enlarging the scope of this bug report

Is my suggestion not valid or doable? I merely presented a visual feedback of a somehow similar design where toolbar and address bar are completely separate from each-other.
